types/ghidra.js/NamespaceManager.d.ts

```diff
@@ -0,0 +1,3 @@
+import type { JavaClass } from './JavaClass';
+/* eslint-disable @typescript-eslint/no-empty-interface */
```

Comment:
`eslint-disable-next-line` instead of enable/disable pairs will clean up a lot of these quite a bit.